### PR TITLE
Ethiopia sky data

### PR DESCRIPTION
**Pull request by mzimmermann-IDM**
### Brief description
Editing file to create data for skyscraper plots (distribution of women by age and parity) from PMA and DHS data for Ethiopia. Code is in data processing folder. Additionally stored a csv of the results in the Ethiopia folder to use for calibration.

### Type(s) of change
- [ ] Bugfix
- [ ] Refactor
- [X ] New feature
- [ ] Other

### Checklist
- My code follows the [style guide](https://github.com/amath-idm/styleguide)
  - [ X] Yes
  - [ ] N/A
- I've commented my code
  - [ X] Yes
  - [ ] N/A
- I've incremented the version number
  - [ X] Yes
  - [ ] N/A
- I've updated the changelog
  - [ ] Yes
  - [X ] N/A
- I've added tests and checked code coverage
  - [ ] Yes
  - [X ] N/A

### CODE REVIEW

**Comment by MObrien-IDM** [comment]
@mzimmermann-IDM Looks great, but I would still like to see the version increased (the minor update seems appropriate!)

**Comment by mzimmermann-IDM** [comment]
Got it, version updated!

File: fpsim/version.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
-__version__ = '0.22.0'
+__version__ = '0.22.2'
 __versiondate__ = '2023-1-24'
```
Comment (MObrien-IDM) [inline fpsim/version.py:2]: Just need to change the date, too!

**Comment by mzimmermann-IDM** [comment]
Oops! Updated.

**Review by MObrien-IDM — APPROVED** [review]

**Pull request merged**